documentation/modules/deploying/proc-deploy-kafka-cluster.adoc

```diff
@@ -22,10 +22,14 @@ For a Kafka deployment, the following examples are provided:
 In this procedure, we use the examples for an _ephemeral_ and _persistent_ Kafka cluster deployment.
 
 Ephemeral cluster:: In general, an ephemeral (or temporary) Kafka cluster is suitable for development and testing purposes, not for production. This deployment uses `emptyDir` volumes for storing broker information (for ZooKeeper) and topics or partitions (for Kafka). Using an `emptyDir` volume means that its content is strictly related to the pod life cycle and is deleted when the pod goes down.
-Persistent cluster:: A persistent Kafka cluster uses `PersistentVolumes` to store ZooKeeper and Kafka data. The `PersistentVolume` is
-acquired using a `PersistentVolumeClaim` to make it independent of the actual type of the `PersistentVolume`. For example, it can use
-ifdef::Kubernetes[HostPath volumes on Minikube or]
-Amazon EBS volumes in Amazon AWS deployments without any changes in the YAML files. The `PersistentVolumeClaim` can use a `StorageClass` to trigger automatic volume provisioning.
+Persistent cluster:: A persistent Kafka cluster uses persistent volumes to store ZooKeeper and Kafka data. A `PersistentVolume` is acquired using a `PersistentVolumeClaim` to make it independent of the actual type of the `PersistentVolume`. The `PersistentVolumeClaim` can use a `StorageClass` to trigger automatic volume provisioning.
++
+A persistent Kafka cluster deployment can use the corresponding type of volume without any changes to the `Kafka` configuration:
```

Comment:
Not sure what was meant by _without any changes in the YAML files_ in the original. I'd like to be specific and added `Kafka` configuration, but I think there might be more to it...